core/include/sched.h

```diff
@@ -174,6 +175,21 @@ extern clist_node_t sched_runqueues[SCHED_PRIO_LEVELS];
  */
 NORETURN void sched_task_exit(void);
 
+#ifdef MODULE_CORE_PRIORITY_INHERITANCE
+/**
+ * @brief   Change the priority of the given thread
+ *
+ * @note    This functions expects interrupts to be disabled when called!
```

Comment:
This could be a `@pre` then, too, right?

Reply:
done